content/docs/for-buildpack-authors/concepts/buildpack-group.md

```diff
@@ -7,7 +6,7 @@ aliases=[
 weight=2
 +++
 
-A buildpack group is a list of individual buildpacks that are designged to work together to build an application.
+A buildpack group is a list of individual buildpacks that are designed to work together to build an application.
```

Comment:
Do we point out here that a Buildpack group used to be called a meta-buildpack?  And that the historical term is still used in some darkened corners of the Internet?

Reply:
@AidanDelaney I didn't actually write this page; just fixed an existing type. However I could add this info.

Reply:
This info is already mentioned in https://buildpacks.io/docs/for-buildpack-authors/concepts/composite-buildpack/
I don't see a need to add it again.